src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -196,12 +232,11 @@ def make_field(
             domain_offset = None
         return gtir_builtin_translators.FieldopData(data_node, data_type, domain_offset)
 
-    def get_symbol_type(self, symbol_name: str) -> ts.DataType:
-        return self.global_symbols[symbol_name]
-
-    def unique_nsdfg_name(self, sdfg: dace.SDFG, prefix: str) -> str:
+    def unique_nsdfg_name(self, prefix: str) -> str:
         nsdfg_list = [
-            nsdfg.label for nsdfg in sdfg.all_sdfgs_recursive() if nsdfg.label.startswith(prefix)
+            nsdfg.label
+            for nsdfg in self.sdfg.all_sdfgs_recursive()
+            if nsdfg.label.startswith(prefix)
         ]
         return f"{prefix}_{len(nsdfg_list)}"
```

Comment:
I do not really like that the get-unique-name-for-a-nested-SDFG step and the add-nested-sdfg are separated.
Is there a way to make this in one step?

Reply:
I could create a wrapper of `add_nested_sdfg()` that changes the name of the nested SDFG, but I don't know if that is a better solution because it would silently change the name of the nested SDFG.